src/common/helsinkiLogo/HelsinkiLogo.tsx

```diff
@@ -9,16 +10,23 @@ type Props = {
 };
 function HelsinkiLogo({ className }: Props) {
   const { t, i18n } = useTranslation();
+  const history = useHistory();
+
+  const navigateToHomePage = () => history.push('/');
 
   if (i18n.languages[0] === 'sv')
     return (
       <span
+        role="button"
```

Comment:
If this element does what a link does, why not have it as an anchor? Is it not semantic?

Reply:
Took another look at the component today and decided to replace spans with `Link` components. I'll just double check before merging that footer is supposed to be link as well.